src/library_browser.js

```diff
@@ -240,28 +240,28 @@ var LibraryBrowser = {
       Module['preloadPlugins'].push(audioPlugin);
 
 #if MAIN_MODULE
-      var wasmPlugin = {};
-      wasmPlugin['asyncWasmLoadPromise'] = new Promise(
-        function(resolve, reject) { return resolve(); });
-      wasmPlugin['canHandle'] = function(name) {
-        return !Module.noWasmDecoding && name.endsWith('.so');
-      };
-      wasmPlugin['handle'] = function(byteArray, name, onload, onerror) {
-        // loadWebAssemblyModule can not load modules out-of-order, so rather
-        // than just running the promises in parallel, this makes a chain of
-        // promises to run in series.
-        this['asyncWasmLoadPromise'] = this['asyncWasmLoadPromise'].then(
-          function() {
-            return loadWebAssemblyModule(byteArray, {loadAsync: true, nodelete: true});
-          }).then(
-            function(module) {
-              Module['preloadedWasm'][name] = module;
-              onload();
-            },
-            function(err) {
-              console.warn("Couldn't instantiate wasm: " + name + " '" + err + "'");
-              onerror();
-            });
+      var wasmPlugin = {
+        asyncWasmLoadPromise: new Promise(function(resolve, reject) { return resolve(); }),
+        canHandle: function(name) {
+          return !Module.noWasmDecoding && name.endsWith('.so')
+        },
+        handle: function(byteArray, name, onload, onerror) {
```

Comment:
These are not strictly equivalent, you need `"handle"` etc., that is, to quote the names here as well. Otherwise closure will minify them.

This may not be an issue as we disable closure in main module mode anyhow. But it would be good to be consistent as perhaps in the future we will want to enable closure there.

Reply:
Oh I see.

In this kind of case would we not want magnification to take effect?     Or maybe there is something special about this code that doesn't want minification?

Reply:
Yes, the use on the other side uses quoted names intentionally (`plugin['canHandle']` in library_fs). That is because a plugin can arrive from outside code in theory, that was not minified together with us. So we want to avoid closure minification here.

Reply:
Gotcha! Sorry I didn't see that.